namespace-specific-policy/automated-rendering/README.md

```diff
@@ -0,0 +1,215 @@
+# Configuring namespace specific policies
+
+One cluster can be shared between multiple tenants. For multi-tenancy, the namespaces used by different tenants should have different policies according to the best practices for multi-tenancy. This doc walks you through the steps for configuring namespace specific policies, such as Role, RoleBinding and NetworkPolicy for a cluster shared between tenants.
+
+## Objectives
+In this tutorial, you will
+- Configure your repository with Kustomize configurations to get namespace specific policies from a common base.
+- Preview and validate the configs that you create.
+- Use Config Sync to automatically render and sync your cluster to your repository.
+- Verify that your installation succeeded.
+
+## Before you begin
+This section describes prerequisites you must meet before this tutorial.
+- ConfigSync is installed on your cluster, with the version at least 1.9.0. If not, you can install
+  it following the [instructions](https://cloud.google.com/anthos-config-management/docs/how-to/installing-config-sync).
+- Install the `nomos` command. If you've already installed `nomos`, make sure you upgrade it to version 1.9.0 or later.
+
+## Create namespace specific policies
+
+### Get the example configuration
+
+The `automated-rendering` directory includes a root `kustomization.yaml` file that references the three tenant overlays.
+
+```yaml
+resources:
+- configsync-src/tenant-a
+- configsync-src/tenant-b
+- configsync-src/tenant-c
+```
+
+The `automated-rendering/configsync-src` file is a symbolic link that links to the source directory: `../configsync-src`.
```

Comment:
Have you verified this example work with the symbolic link used here?

Reply:
Yes, the `automated-rendering` directory can be synced successfully to a cluster with the symbolic link.

Reply:
Thanks for confirming.